packages/fe-container/src/utils/plugins/PluginsLoaderFacade.test.ts

```diff
@@ -149,6 +154,38 @@ describe('Test plugin loading', () => {
     }])
   })
 
+  it('test qiankun with shared props', () => {
```

Comment:
Can you test the scenario in which if you define the same property for the plugin and for the shared props, the value inside the plugin wins?

Reply:
added test with description: 
`test qiankun with both shared and plugin props => plugin props must override shared`